lib/David_tic_tac_toe.rb

```diff
@@ -1,9 +1,6 @@
 # frozen_string_literal: true
 
-require_relative 'David_tic_tac_toe/version'
-
-module DavidTicTacToe
-  def self.print
-    puts 'Welcome to Tic Tac Toe Game using console'
-  end
-end
+# require_relative 'David_tic_tac_toe/version'
+# require_relative 'David_tic_tac_toe/board'
+# require_relative 'David_tic_tac_toe/player'
+# require_relative 'David_tic_tac_toe/game'
```

Comment:
Since everything is commented out in this file, I wonder if we can delete it?

Reply:
Tried to delete it but my tests failed
    _It says problems with spec_helper_ I am going do research about the spec_helper though